ml3d/datasets/kitti.py

```diff
@@ -0,0 +1,261 @@
+import numpy as np
+import os, argparse, pickle, sys
+from os.path import exists, join, isfile, dirname, abspath, split
+from pathlib import Path
+from glob import glob
+import logging
+import yaml
+
+from ..utils import Config, make_dir, DATASET
+
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(levelname)s - %(asctime)s - %(module)s - %(message)s',
+)
+log = logging.getLogger(__name__)
+
+
+class KITTI():
```

Comment:
will it have a base class?

Reply:
Done. The semseg base class works. If needed, we can add separate base class in future.